File: remove_comments_temp.js
const fs = require('fs');
const path = require('path');
const file = path.join(__dirname, 'frontend', 'src', 'pages', 'userPage', 'HomeWithAuth.tsx');
let text = fs.readFileSync(file, 'utf8');
text = text.replace(/\/\/.*$/gm, '');
text = text.replace(/\/\*[\s\S]*?\*\//g, '');
text = text.replace(/<!--([\s\S]*?)-->/g, '');
fs.writeFileSync(file, text, 'utf8');
console.log('removed comments from HomeWithAuth.tsx');

File: remove_comments_temp.py
import pathlib
import re
path = pathlib.Path(r"c:\Users\Asus\Desktop\mern-app\frontend\src\pages\userPage\HomeWithAuth.tsx")
text = path.read_text(encoding='utf-8')
text = re.sub(r'//.*?$','', text, flags=re.MULTILINE)
text = re.sub(r'/\*.*?\*/','', text, flags=re.DOTALL)
text = re.sub(r'<!--.*?-->','', text, flags=re.DOTALL)
path.write_text(text, encoding='utf-8')
print('removed comments from HomeWithAuth.tsx')
